### PR TITLE
Created Timestamp for Post

### PR DESCRIPTION
**Pull request by revett**
### Problem

The UI requires the datetime of when the post was created.

### Solution

- Add functions for new API call
- Add functions to parse result of API call

### Notes

- None

### CODE REVIEW

**Comment by revett** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;DfNtNBhW&#x2F;339-fbc-get-published-timestamp-on-each-post"> 👴 [FBC] Get published timestamp on each Post</a>

File: post.go [hunk 1]
```diff
@@ -40,11 +41,16 @@ func (p Post) GenerateInsightParams() BatchParams {
...
 }
 
-// GenerateParams comments pending
+// GenerateParams comment pending
 func (p *Post) GenerateParams() BatchParams {
 	return NewBatchParams(fmt.Sprintf("%s?fields=%s", p.ID, "object_id,message"))
 }
 
+// GeneratePostCreatedTimestampParams comment pending
+func (p *Post) GeneratePostCreatedTimestampParams() BatchParams {
```
Comment (stevenjack) [inline post.go]: Do we need to do __another__ call to the post endpoint.. can't we just add `created_time` to the list of field in `GenerateParams()` then we'll also have the `created_time` in that payload..

Comment (revett) [inline post.go]: I wanted to avoid putting more and more into the FBP to FBC payload. Thoughts?

Comment (stevenjack) [inline post.go]: I guess this is what makes it a bit weird/confusing how some parts are used by one app and some by the other. Makes sense now as I think `FBP` uses `GenerateParams()` and `FBC` will be using `GeneratePostCreatedTimestampParams()`. Ignore this comment 👍

**Comment by stevenjack** [comment]
:cake:

File: post.go (hunk 1 above)
Comment (revett) [inline post.go]: 🙆

**Pull request merged**